index.js

```diff
@@ -3,31 +3,48 @@ const bookAuthor = document.getElementById('bookAuthor');
 const bookForm = document.getElementById('bookForm');
 const bookRec = document.getElementById('books-record');
 
+const booksrecord = JSON.parse(localStorage.getItem('bookArchive')) || [];
+
 class Book {
   constructor(title, author) {
     this.title = title;
     this.author = author;
     this.id = new Date().getTime().toString().concat(performance.now());
   }
-}
 
-const booksrecord = JSON.parse(localStorage.getItem('bookArchive')) || [];
+  static displayBook(title, author, id) {
+    const templateHTML = `
+      <tr>
+        <td class="table-item">
+          "${title}" by ${author}
+          <button type='button' class="remove-bttn" id="${id}">Remove</button>
+        </td>
+      </tr>`;
 
-function displayBook(title, author, id) {
-  const templateHTML = `
-    ${title}<br>
-    ${author}<br>
-    <button type='button' class="remove-bttn" id="${id}">Remove</button>
-    <hr>`;
+    bookRec.insertAdjacentHTML('beforeend', templateHTML);
+  }
 
-  bookRec.insertAdjacentHTML('beforeend', templateHTML);
-}
+  static getBooks() {
+    const storage = JSON.parse(localStorage.getItem('bookArchive'));
+    storage.forEach((book) => {
+      Book.displayBook(book.title, book.author, book.id);
+    });
+  }
 
-function getBooks() {
-  const storage = JSON.parse(localStorage.getItem('bookArchive'));
-  storage.forEach((book) => {
-    displayBook(book.title, book.author, book.id);
-  });
+  static remove(button, key) {
+    button.addEventListener('click', () => {
+      if (key === 0) {
+        booksrecord.splice(key, key + 1);
+      } else {
+        booksrecord.splice(key, key);
+      }
+
+      bookRec.innerHTML = '';
+      localStorage.setItem('bookArchive', JSON.stringify(booksrecord));
+      window.location.reload();
+      localStorage.setItem('bookStorage', JSON.stringify(booksrecord));
+    });
+  }
 }
 
 function insertBook() {
```

Comment:
- [ ] I see that the `insertBook()` is the function that is responsible for adding books. I would recommend adding this function `insertBook()` to the Books class as a class method so your code matches the project requirements.